src/libs/utils.js

```diff
@@ -185,6 +185,8 @@ export const abandonedPromise = () => {
 
 export const generateClusterName = () => `saturn-${uuid()}`
 
+export const generateKubernetesClusterName = () => `saturn-k8-${uuid()}`.slice(0, 25)
```

Comment:
Is there a shorter form of UUID we can use instead of arbitrarily chopping off the end of this one? Maybe we can remove the `-`s? I know we're unlikely to see any collisions... probably ever... but if someone does, it will be practically impossible to reproduce and really hard to recognize what happened.

Reply:
Ah, this should actually be fine instead of chopping off. Will update